src/components/v5/shared/VoteChart/VoteChart.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+import { useIntl } from 'react-intl';
+import { VoteChartProps } from './types';
+import VoteChartBar from './partials/VoteChartBar';
+import { VOTE_CHART_BAR_DIRECTION } from './partials/VoteChartBar/types';
+import VoteChartThresholdIndicator from './partials/VoteChartThresholdIndicator';
+
+const VoteChart: FC<VoteChartProps> = ({
+  percentageVotesFor = 0,
+  forLabel,
+  percentageVotesAgainst = 0,
+  againstLabel,
+  threshold,
+  thresholdLabel,
+}) => {
+  const intl = useIntl();
+
+  const forValue =
+    (percentageVotesFor < 0 && 0) ||
+    (percentageVotesFor > 100 && 100) ||
+    percentageVotesFor;
+
+  const againstValue =
+    (percentageVotesAgainst < 0 && 0) ||
+    (percentageVotesAgainst > 100 && 100) ||
+    percentageVotesAgainst;
+
+  return (
+    <div className="w-full">
+      <p className="text-xs font-medium text-center mb-1 text-blue-400">
+        {thresholdLabel ||
+          intl.formatMessage(
+            { id: 'motion.staking.threshold.label' },
+            { threshold: `${threshold}%` },
+          )}
+      </p>
+      <div className="flex pt-2">
+        <div className="flex flex-1 flex-col items-center gap-1">
+          <VoteChartBar
+            value={againstValue}
+            barBackgroundClassName="bg-red-300"
+            direction={VOTE_CHART_BAR_DIRECTION.Left}
+          />
+          <span
+            className={clsx('text-xs text-center transition', {
```

Comment:
Do you need transition here?

Reply:
Yes, I want to animate text color change